### ISSUE TITLE
Page link functionality

### ISSUE THREAD

**Issue by medienbaecker** [JonasDoebertin/kirby-visual-markdown#71]
I recently added a page link functionality to the Visual Markdown Editor. So you don't have to search for the URL-path for every page you want to link. This functionality exists in most CMS I previously used. Some of my clients were visibly disappointed that this is not so easy. Most of them came from Typo3 or Wordpress.
I'm still pretty sure this is not going to make it into the core Kirby textarea though. But maybe it makes sense to include it in the Visual Markdown Editor? I use it in almost every client website, anyway.
What do you think?

![pagelink-explanation](https://cloud.githubusercontent.com/assets/7975568/11978363/510b148c-a98b-11e5-859f-afe6f9a18920.gif)

Here's a link to my fork: https://github.com/medienbaecker/kirby-visual-markdown

**Comment by JonasDoebertin** [comment]
Looks pretty sweet! I'd love to merge it into Visual Markdown Editor!

**Comment by medienbaecker** [comment]
Thanks!
We will then need the translations for "Choose page" or "Seite auswählen". And maybe a tooltip saying "Insert page link" or "Seitenlink einfügen"?
By the way, I changed the behavior of the modals slightly in my fork. The keyboard shortucts were never higher than the window height, but the page link modal is most of the time. So I added a 98% max-height.
Should I request a pull to the "develop" branch?
